.github/workflows/coding-standards.yml

```diff
@@ -0,0 +1,101 @@
+name: PHP Coding Standards
+
+on:
+    push:
+        branches:
+            - trunk
+            - 'release/**'
+            - 'wp/**'
+    pull_request:
+        paths:
+            # Any change to a PHP file should run checks.
+            - '**.php'
+            # These files configure Composer. Changes could affect the outcome.
+            - 'composer.*'
+            # This file configures PHPCS. Changes could affect the outcome.
+            - 'phpcs.xml.dist'
+            # Changes to workflow files should always verify all workflows are successful.
+            - '.github/workflows/*.yml'
```

Comment:
Please add:
```yml
  # Allow manually triggering the workflow.
  workflow_dispatch:
```

Manual triggering is not allowed by default in GHA, so needs to be explicitly permitted.

I've found this useful on numerous occasions, like when a bug was fixed in an action runner, a new Composer version was released, a new PHP patch version was released etc.

Reply:
Sure. Thanks for pointing it out.
Fixed in d448a66025a438b8a318c1711159f98043053c16